crates/goose-cli/src/commands/configure.rs

```diff
@@ -0,0 +1,74 @@
+use cliclack::input;
+use console::style;
+use ctrlc;
+use serde::{Deserialize, Serialize};
+use std::error::Error;
+use std::fs::{create_dir_all, File};
+use std::io::Write;
+use std::path::PathBuf;
+
+#[derive(Serialize, Deserialize)]
+pub struct ConfigOptions {
+    pub provider: Option<String>,
+    pub host: Option<String>,
+    pub token: Option<String>,
+    pub processor: Option<String>,
+    pub accelerator: Option<String>,
+}
+
+pub fn handle_configure(options: ConfigOptions) -> Result<(), Box<dyn Error>> {
+    ctrlc::set_handler(move || {}).expect("setting Ctrl-C handler");
+    
+    cliclack::clear_screen()?;
+
+    cliclack::intro(style(" create-app ").on_cyan().black())?;
+    let provider = prompt(options.provider, "Enter provider name:");
+    let host = prompt(options.host, "Enter host URL:");
+    let token = prompt(options.token, "Enter token:");
+    let processor = prompt(options.processor, "Enter processor:");
+    let accelerator = prompt(options.accelerator, "Enter accelerator:");
+
+    let final_config = ConfigOptions {
+        provider: Some(provider),
+        host: Some(host),
+        token: Some(token),
+        processor: Some(processor),
+        accelerator: Some(accelerator),
+    };
+    match save_to_yaml(&final_config) {
```

Comment:
imo would be good to test this here by calling provider.complete() to confirm it works and share a fun message